packages/braintree-integration/src/braintree-paypal/create-braintree-paypal-payment-strategy.ts

```diff
@@ -9,7 +9,8 @@ import {
     PaymentStrategyFactory,
     toResolvableModule,
 } from '@bigcommerce/checkout-sdk/payment-integration-api';
-import { Overlay } from '@bigcommerce/checkout-sdk/ui';
+import { LOADING_INDICATOR_STYLES } from '@bigcommerce/checkout-sdk/paypal-commerce-utils';
```

Comment:
We should not use PayPal Commerce specific code in Braintree integration. We have 2 ways to solve this issue:
1) move `LOADING_INDICATOR_STYLES` into `ui` package and use it from there (but leave ppcp refactoring for another PR);
2) make a copy of `LOADING_INDICATOR_STYLES` and paste it somewhere in braintree related package;